stack.yaml

```diff
@@ -14,27 +14,31 @@ extra-deps:
     https://gitlab.com/morley-framework/morley.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    7e81317ef21f86d17f04b63c1c9679b710fdb54b # master
-    #nix-sha256: 0yy1r9w0jlj11fpfk02p6pn53fcvc4djqflsbd8srz5ngndc57pi
+    e6fa4c2b98dcd31ffafd307dc204cc937a070b99 # TODO: not master
+    #nix-sha256: ?
   subdirs:
     - code/indigo
     - code/lorentz
     - code/morley
```

Comment:
I propose to take `indigo`, `lorentz` and `morley` from Hackage, they are relatively stable lately (especially `morley`). This way we can sometimes avoid unnecessary rebuilds when working on multiple projects.

Reply:
I'm using a super recent commit of morley (because of one recent merged MR), so this may be complicated :pled:

Reply:
If it's about `MorleyClientConfig` lenses, it shouldn't matter because they were added to `morley-client` that you are taking from git anyway. `morley` and `lorentz` didn't change since latest release in a way that can affect you, I think.

Reply:
Oh right, fair. Updated.